solutions/usaco-809.mdx

```diff
@@ -0,0 +1,69 @@
+---
+id: usaco-809
+source: USACO Bronze 2018 Febuary
+title: Taming the Herd
+author: Sathvik Chundru
+---
+
+[Official Analysis (C++)](http://www.usaco.org/current/data/sol_taming_bronze_feb18.html)
+
+**Time Complexity:** $\mathcal{O}(N)$
+
+<LanguageSection>
+
+<JavaSection>
+	
+```java
+import java.util.*;
+import java.io.*;
+
+public class Taming{
+	public static void main(String[] args) throws IOException{
+		Kattio io = new Kattio("taming");
+		int n = io.nextInt();
+		int[] log = new int[100000];
+		for( int i = 0; i < n; i++ ){
+			log[i] = io.nextInt();
+		}
+		if( log[0] > 0 ){
+			io.println(-1);
+			io.close();
+			System.exit(0);
+		}
+		log[0] = 0;
+		int t = -1;
+		int req = 0;
+		int pos = 0;
+		for( int i = n - 1; i >= 0; i-- ){
```

Comment:
:notlikethis:
i mean something like
```java
for (int i = n - 1; i >= 0; i--) {
}
```

Reply:
fixed

Reply:
did you not see the spaces here
![image](https://user-images.githubusercontent.com/55369003/127713218-563fdfd1-d0e7-4955-8369-1a4c9e820624.png)
![image](https://user-images.githubusercontent.com/55369003/127713225-c46cc1e6-f219-43be-a3a4-dfa1a51ae3c5.png)

Reply:
fixed